host/http.go

```diff
@@ -42,16 +41,8 @@ func (h *Host) streamEvents(id string, w http.ResponseWriter) error {
 		<-w.(http.CloseNotifier).CloseNotify()
```

Comment:
I think an explicit `Error` or `CloseWithError` on the stream is better than using a `chan interface{}`.